src/main/java/tri/stringreverse/StringReverse.java

```diff
@@ -0,0 +1,16 @@
+package tri.stringreverse;
+
+/**
+ * Output a reversed string.
+ */
+public class StringReverse {                //create class
+    public String reverse(String text){     //function named reverse, the function used a built-in reverse function and return the reversed string
+        return reverse(text);
+    }
+
+    public static void main(String[] args) {
+        StringReverse goal = new StringReverse();
+        System.out.println(goal.reverse());
```

Comment:
The goal.reverse() do not compile because the method reverse require a text. By the way, even though it's not required in the assignment, but since you asked, you can also prompt for user input for a string, and display the result. To catch a user input, you can use a Scanner:
```java
        System.out.print("Enter your text: ");
        Scanner scanner = new Scanner(System.in);
        String inputText = scanner.nextLine();
```